tf/edgeml/trainer/bonsaiTrainer.py

```diff
@@ -37,6 +38,11 @@ def __init__(self, bonsaiObj, lW, lT, lV, lZ, sW, sT, sV, sZ,
 
         self.useMCHLoss = useMCHLoss
 
+        if outFile is not None:
```

Comment:
This is not needed actually. If file is `None` print will default to `sys.stdout` or whatever the default output stream is. Though there is no harm in doing this so its fine by me.

Reply:
@metastableB Yes, but .flush() method works on sys.stdout and not on None.
I need to ensure that it happens.
We need to flush every epoch stats immediately to the files, which is not done by default, thus had to do it this way.